includes/widgets/heading.php

```diff
@@ -427,7 +426,7 @@ protected function render() {
 
 		$this->add_inline_editing_attributes( 'title' );
 
-		$title = wp_kses_post( $settings['title'] );
+		$title = ( ( is_admin() && ! current_user_can( 'manage_options' ) ) || ! empty( $settings['isShortcode'] ) ) ? wp_kses_post( $settings['title'] ) : $settings['title'];
```

Comment:
Can we extract `is_admin() && ! current_user_can( 'manage_options' ) ) || ! empty( $settings['isShortcode'] ` to a seperate function to increase code readability?